golangci-lint-config.yaml

```diff
@@ -2,77 +2,79 @@
 
 run:
   deadline: 240s
-  tests: true
   skip-dirs:
     - vendor
-  build-tags: acceptance
+    - docs
+    - build
 
 linters-settings:
+  exhaustive:
+    default-signifies-exhaustive: true
+
+  wsl:
+    force-case-trailing-whitespace: 1
+
   dupl:
     threshold: 110
+
   lll:
     line-length: 128
-  goimports:
-    local-prefixes: 'github.com/hpe-hcss'
-  godox:
-    keywords:
-      - OPTIMIZE
-  exhaustive:
-    default-signifies-exhaustive: true
-  funlen:
-    lines: 90
-    statements: 60
+    # tab width in spaces. Default to 1.
+    tab-width: 1
+
+issues:
+  new-from-rev: origin/master
```

Comment:
This may cause issues if people are creating branches from branches not master. In $QUAKE, I don't specify new-from-rev in the config file, I pass in a calculated value from the build script.

Reply:
could you please point me to the script?

Reply:
I think this is still OK as long as it is compared against the master as baseline

Reply:
I don't know how well this will translate to GitHub Actions since $QUAKE runs under CircleCi: https://github.com/hpe-hcss/quake/blob/master/.circleci/config.yml#L683. The main 2 points are first get the correct target branch and not assume `master`. Also, use `git merge-base` to get a cleaner rev to merge against.

Reply:
Ok, thanks, followed the same scripts here as well in the latest

Reply:
Since you added `new-from-rev` to the command line, is this `issues` section still needed in the config?

Reply:
keeping it so that when 'make lint' runs, this is considered.

Reply:
removed from here, added in Makefie directly